packages/test-helpers/src/bin/github-comment-manager.test.js

```diff
@@ -0,0 +1,181 @@
+/* global describe it expect */
+import githubCommentManager from "./github-comment-manager";
+
+const nock = require("nock");
+
+const account = "testAccountName";
+const token = "testKey";
+const repository = "testRepo";
+const pullRequest = 1;
+const documentPath = "index.html";
+
+const noCommentsResponse = [
+  {
+    id: 1,
+    user: {
+      login: "not test account name"
+    }
+  }
+];
+
+const mixedCommentsResponse = [
+  {
+    id: 1,
+    body: "If you use Expo, view our components by scanning this qr code:",
+    user: {
+      login: "not test account name"
+    }
+  },
+  {
+    id: 2,
+    body: "If you use Expo, view our components by scanning this qr code:",
+    user: {
+      login: account
+    }
+  },
+  {
+    id: 3,
+    body: "If you use Expo, view our components by scanning this qr code:",
+    user: {
+      login: "a different non test account name"
+    }
+  }
+];
+
+const multipleUserCommentsResponse = [
+  {
+    id: 1,
+    body: "If you use Expo, view our components by scanning this qr code:",
+    user: {
+      login: account
+    }
+  },
+  {
+    id: 2,
+    body: "If you use Expo, view our components by scanning this qr code:",
+    user: {
+      login: account
+    }
+  }
+];
+
+const sameUserCommentsResponse = [
+  {
+    id: 1,
+    body: "Please find visual snapshots of your changed components here:",
+    user: {
+      login: account
+    }
+  },
+  {
+    id: 2,
+    body: "If you use Expo, view our components by scanning this qr code:",
+    user: {
+      login: account
+    }
+  }
+];
+
+describe("deleteAllExpoComments", () => {
+  it("should not delete any comments", async () => {
```

Comment:
Why would it not delete any comments? This test name needs to be more descriptive.